public/app/plugins/datasource/grafana-pyroscope-datasource/VariableQueryEditor.tsx

```diff
@@ -65,6 +65,8 @@ export function VariableQueryEditor(props: QueryEditorProps<PyroscopeDataSource,
               props.onChange({ ...props.query, profileTypeId: val });
             }
           }}
+          from={props.range?.from.valueOf() || Date.now().valueOf() - 1000 * 60 * 60 * 24}
```

Comment:
Not sure why we're going back a day here but providing start/end of 0 in `<QueryEditor />`?

Reply:
I copy-pasted this from the `<LabelRow />` part, assuming we wanted similar behavior.

https://github.com/grafana/grafana/blob/8e01744e32e0c32ff3a2f68b8d526d307a22f43a/public/app/plugins/datasource/grafana-pyroscope-datasource/VariableQueryEditor.tsx#L74-L85